spec/features/save_document_associations_api_down_spec.rb

```diff
@@ -13,11 +13,12 @@
   end
 
   def given_there_is_a_document_with_associations
-    @document = create(:document, :with_associations_in_schema)
-
-    @document.document_type_schema.associations.each do |schema|
-      publishing_api_has_linkables([], document_type: schema.document_type)
-    end
+    association_schema = attributes_for(:association, type: "multi_association")
+    document_type_schema = build(:document_type_schema, associations: [association_schema])
+    association = { association_schema[:id] => ["a-content-id"] }
+    linkable = { "content_id" => ["a-content-id"], "internal_name" => "Association" }
```

Comment:
We don't actually need this - can just pass `[]` to the stub, as we're going to take it down anyway and doing the request doesn't depend on the select box having any items.